### PR TITLE
Add deploy-8 docs

### PR DESCRIPTION
**Pull request by gomathyk**
Deploy/security (except for encryption)

### CODE REVIEW

File: en/docs/deploy/security/customize-secure-vault.md [hunk 1]
```diff
@@ -46,40 +45,36 @@ sample, you do not need to configure a Secret Repository or keystore as you are
...
 
-3.  Create a JAR or an OSGI bundle and copy the JAR file to the
+3.  Create a JAR or an OSGI bundle and copy it to the
     `IS_HOME/repository/component/lib/` directory or the OSGI bundle to the
-    `IS_HOME/repository/component/dropins/directory`
-    .
-4.  Configure the ei.toml file
-    with an alias name and your Secret Callback handler class name. For
-    example,  
+    `IS_HOME/repository/component/dropins/directory` respectively.
```
Comment (dewniMW) [inline en/docs/deploy/security/customize-secure-vault.md:75]:
```suggestion
    `<IS_HOME>/repository/component/dropins/directory` respectively.
```

File: en/docs/deploy/security/customize-secure-vault.md [hunk 2]
```diff
@@ -129,7 +123,4 @@ To create a custom secret repository, you need to implement the
 
 3.  Create a JAR or an OSGI bundle.
 
-4.  Then, copy the JAR file to the
-    `IS_HOME/repository/component/lib/`
-    directory or the OSGI bundle to the
-    `IS_HOME/repository/component/dropins/` directory.
\ No newline at end of file
+4.  Copy the JAR file to the `IS_HOME/repository/component/lib/` directory or the OSGI bundle to the `IS_HOME/repository/component/dropins/` directory.
```
Comment (dewniMW) [inline en/docs/deploy/security/customize-secure-vault.md:126]:
```suggestion
4.  Copy the JAR file to the `<IS_HOME>/repository/component/lib/` directory or the OSGI bundle to the `<IS_HOME>/repository/component/dropins/` directory.
```

File: en/docs/deploy/security/enable-hsts.md [hunk 3]
```diff
@@ -1,18 +1,17 @@
...
 !!! note 
-
 	HSTS should not be enabled in development environments because transport security validations can interrupt the development processes by validating signatures of self-signed certificates. 
 
 ---
 
 ## Enable for Management Console 
 
-If the `HttpHeaderSecurityFilter` element is available in the `web.xml` file (stored in the `<IS_HOME>/repository/conf/tomcat/carbon/WEB-INF/` directory) as shown below, it implies that security headers are by default configured for the management consoles of all of your profiles. However, in a production deployment, Strict-Transport-Security needs to be explicitly enabled by replacing the default <init-param> values of the `HttpHeaderSecurityFilter` filter.
+If the `HttpHeaderSecurityFilter` element is available in the `<IS_HOME>/repository/conf/tomcat/carbon/WEB-INF/web.xml` file as shown below, it implies that security headers are by default configured for the WSO2 Management Consoles of all of your profiles. However, in a production deployment, Strict-Transport-Security needs to be explicitly enabled by replacing the default <init-param> values of the `HttpHeaderSecurityFilter` filter.
```
Comment (dewniMW) [inline en/docs/deploy/security/enable-hsts.md:12]: `<init-param>` -> is this shown as expected in the UI?

File: en/docs/deploy/security/enable-hsts.md [hunk 4]
```diff
@@ -62,22 +61,22 @@ Similar to the management console, check whether the `HttpHeaderSecurityFilter`
 
 ## Enable for Jaggery applications
```
Comment (dewniMW) [inline en/docs/deploy/security/enable-hsts.md:62]: This section talks about the old dashboard

File: en/docs/deploy/security/enable-java-security-manager.md [hunk 5]
```diff
@@ -18,22 +18,19 @@ When granting specific Java-level permissions to a certain signed code, it is re
...
+1. 	To sign the JARs in WSO2 Identity Server, a **key** is required. You can generate a key by generating a new keystore. Follow the steps given below to generate a new keystore with a new key.
 
 	1. Execute the following keytool command. 
 
 		!!! note
 			The new keystore is created in the directory in which you execute the keytool command. 
 
 	    ```
-	    keytool -genkey -alias signFiles -keyalg RSA -keystore signkeystore.jks -validity 3650 -dname "CN=mark,OU=Engineering, O=WSO2, L=Colombo, ST=Western, C=LK"
+	    keytool -genkey -alias signFiles -keyalg RSA -keystore signkeystore.jks -validity 3650 -dname "CN=mark,OU=Engineering, O=WSO2, L=Colombo, ST=Western, C=LK"5
```
Comment (dewniMW) [inline en/docs/deploy/security/enable-java-security-manager.md:33]: why is this change required?

File: en/docs/deploy/security/encrypt-passwords-with-cipher-tool.md [hunk 6]
```diff
@@ -63,7 +63,7 @@ password = "$secret{keystore_password}"
 ```
 
 ### Passwords in LOG4j properties
-For example, consider the 'log4j.appender.LOGEVENT.password' in the log4j.properties file. You can refer the [encrypted password](#encrypt-passwords) from the log4j.properties file as shown below.
+For example, consider the 'log4j.appender.LOGEVENT.password' in the `log4j.properties` file. You can refer the [encrypted password](#encrypt-passwords) from the `log4j.properties` file as shown below.
```
Comment (dewniMW) [inline en/docs/deploy/security/encrypt-passwords-with-cipher-tool.md:66]: in the new IS versions we are having log4j2.properties file

File: en/docs/deploy/security/encrypt-passwords-with-cipher-tool.md [hunk 7]
```diff
@@ -73,10 +73,10 @@ log4j.appender.LOGEVENT.password=secretAlias:log4j.appender.LOGEVENT.password
 
 ## Change encrypted passwords
 
-To change any password that we have encrypted already, follow the below steps:
+To change any password that we has been encrypted already, follow the steps given below.
```
Comment (dewniMW) [inline en/docs/deploy/security/encrypt-passwords-with-cipher-tool.md:76]:
```suggestion
To change any password that has been encrypted already, follow the steps given below.
```

File: en/docs/deploy/security/encrypt-passwords-with-cipher-tool.md [hunk 8]
```diff
@@ -85,8 +85,8 @@ To change any password that we have encrypted already, follow the below steps:
...
 7. The system will then prompt you (twice) to enter the new password. Enter your new password.
 
-!!! info
+!!! info "Related topics"
     
-    For information on resolving the encrypted passwords, see [Resolving Encrypted Passwords](../../../deploy/security/resolve-encrypted-passwords).
+    [Deploy: Resolve Encrypted Passwords](../../../deploy/security/resolve-encrypted-passwords)
 
-    For information on customizing secure vault implementaion, see [Customizing Secure Vault](../../../deploy/security/customize-secure-vault).
+    [Deploy: Customize Secure Vault](../../../deploy/security/customize-secure-vault)
```
Comment (dewniMW) [inline en/docs/deploy/security/encrypt-passwords-with-cipher-tool.md:92]: are these displayed as bullet points?

File: en/docs/deploy/security/maintain-logins-and-passwords.md [hunk 9]
```diff
@@ -2,7 +2,7 @@
 
 ## Change the super admin password
 
-To change the default admin password, log in to the management console
+To change the default admin password, log in to the WSO2 management console (`https://<IS_HOST>:<PORT>/carbon`)
```
Comment (dewniMW) [inline en/docs/deploy/security/maintain-logins-and-passwords.md:5]: `WSO2 management console` or `WSO2 Identity Server Management Console`?

File: en/docs/deploy/security/prevent-browser-caching.md [hunk 10]
```diff
@@ -31,18 +31,19 @@ You can enable these headers for the management console by adding the following
...
-Note that the applications that are included in the `/webapps` directory by default in a WSO2 product do not serve sensitive content that requires cache prevention. However, if you are adding any new applications, you need to be mindful of this requirement.
+Note that the applications that are included in the `/webapps` directory by default in WSO2 Identity Server do not serve sensitive content that requires cache prevention. However, if you are adding a new application, you need to be mindful of this requirement.
 
 ---
 
 ## Prevent for Jaggery applications
 
-For Jaggery-based applications (stored in the `<IS_HOME>/repository/deployment/server/jaggeryapps/` directory), either `URLBasedCachePreventionFilter` or `ContentTypeBasedCachePreventionFilter` should be available in the `jaggery.conf` file as shown below.
-
+For Jaggery-based applications stored in the `<IS_HOME>/repository/deployment/server/jaggeryapps/` directory), either `URLBasedCachePreventionFilter` or `ContentTypeBasedCachePreventionFilter` should be available in the `jaggery.conf` file as shown below.
```
Comment (dewniMW) [inline en/docs/deploy/security/prevent-browser-caching.md:42]: check if this directory is there in latest IS

File: en/docs/deploy/security/product-level-security-guidelines.md [hunk 11]
```diff
@@ -102,16 +98,15 @@ The TLS protocol and ciphers are configured for an HTTP connector using the `SSL
...
 
 ---
 
 ## HSTS
 
 For products based on Carbon 4.4.11 or later versions, HTTP Strict Transport Security (HSTS) is disabled for the applications with which WSO2 Identity Server is shipped by default. This is because HSTS validation can interrupt the development processes by validating signatures of self-signed certificates.
 
-
-Make sure to enable  (HSTS) for all the applications that are deployed in your WSO2 Identity Server. This includes Management Console and any other web applications. For instructions, see <a href="../../../deploy/security/enable-hsts">Enabling HTTP Strict Transport Security (HSTS) Headers</a>.
+Make sure to enable (HSTS) for all the applications that are deployed in WSO2 Identity Server. This includes the WSO2 Identity Server Management Console and any other web applications. For instructions, see <a href="../../../deploy/security/enable-hsts">Enable HTTP Strict Transport Security (HSTS) Headers</a>.
```
Comment (dewniMW) [inline en/docs/deploy/security/product-level-security-guidelines.md:109]:
```suggestion
Make sure to enable HSTS for all the applications that are deployed in WSO2 Identity Server. This includes the WSO2 Identity Server Management Console and any other web applications. For instructions, see <a href="../../../deploy/security/enable-hsts">Enable HTTP Strict Transport Security (HSTS) Headers</a>.
```

File: en/docs/deploy/security/product-level-security-guidelines.md [hunk 12]
```diff
@@ -227,38 +216,38 @@ If required, increase the `session ID length` by changing the `sessionIDLength`
...
+Make sure that the permission for signing in to the WSO2 Identity Server Management Console is granted only to the users that need to use the Management Console. Instead of granting all permission to one administrator, distribute the responsibilities among multiple administrators by assigning different permissions. For instructions, see <a href="../../../guides/identity-lifecycles/manage-roles-overview/">Manage User Roles</a>.
 
 ---
 
 ## Log rotation and monitoring
 
-Ensure that you have a relevant log rotation scheme to manage logs. Log4J properties for WSO2 Identity Server can be configured in the `log4j.properties` file in the `<IS_HOME>/repository/conf` file. 
+Ensure that you have a relevant log rotation scheme to manage logs. Log4J properties for WSO2 Identity Server can be configured in the `log4j.properties` file in the `<IS_HOME>/repository/conf` folder.
```
Comment (dewniMW) [inline en/docs/deploy/security/product-level-security-guidelines.md:248]: new IS versions have log4j2.properties

File: en/docs/deploy/security/product-level-security-guidelines.md [hunk 13]
```diff
@@ -227,38 +216,38 @@ If required, increase the `session ID length` by changing the `sessionIDLength`
...
 ## Log rotation and monitoring
 
-Ensure that you have a relevant log rotation scheme to manage logs. Log4J properties for WSO2 Identity Server can be configured in the `log4j.properties` file in the `<IS_HOME>/repository/conf` file. 
+Ensure that you have a relevant log rotation scheme to manage logs. Log4J properties for WSO2 Identity Server can be configured in the `log4j.properties` file in the `<IS_HOME>/repository/conf` folder. 
 
-To roll the `wso2carbon.log` based on the size, use the following configurations:
+To roll the `wso2carbon.log` based on the size, use the following configurations.
 
 ```java 
 log4j.appender.CARBON_LOGFILE=org.apache.log4j.RollingFileAppender
```
Comment (dewniMW) [inline en/docs/deploy/security/product-level-security-guidelines.md:253]: will need to check the way how it is in the new file

File: en/docs/deploy/security/product-level-security-guidelines.md [hunk 14]
```diff
@@ -279,15 +268,15 @@ Log forging can be prevented by appending a UUID to the log message.
 
 !!! info
 
-    For more information on configuring the `log4j.properties` file, see <a href="../../../deploy/monitor/monitor-logs/">Monitoring Logs</a>.
+    For more information on configuring the `log4j.properties` file, see <a href="../../../deploy/monitor/monitor-logs/">Monitor Logs</a>.
```
Comment (dewniMW) [inline en/docs/deploy/security/product-level-security-guidelines.md:271]: `log4j2.properties`

File: en/docs/deploy/security/enable-hsts.md (hunk 3 above)
Comment (gomathyk) [reply en/docs/deploy/security/enable-hsts.md:12]: Yes

File: en/docs/deploy/security/enable-hsts.md (hunk 4 above)
Comment (gomathyk) [reply en/docs/deploy/security/enable-hsts.md:62]: Commented it out. Please confirm if this section is only used for the dashboard functionality in which case we might have to hide the whole section.

File: en/docs/deploy/security/encrypt-passwords-with-cipher-tool.md [hunk 15]
```diff
@@ -63,7 +63,7 @@ password = "$secret{keystore_password}"
 ```
 
 ### Passwords in LOG4j properties
-For example, consider the 'log4j.appender.LOGEVENT.password' in the log4j.properties file. You can refer the [encrypted password](#encrypt-passwords) from the log4j.properties file as shown below.
+For example, consider the 'log4j.appender.LOGEVENT.password' in the `log4j2.properties` file. You can refer the [encrypted password](#encrypt-passwords) from the `log4j2.properties` file as shown below.
```
Comment (dewniMW) [inline en/docs/deploy/security/encrypt-passwords-with-cipher-tool.md:66]: `log4j.appender.LOGEVENT.password` -> is this there in the new file?

File: en/docs/deploy/security/product-level-security-guidelines.md [hunk 16]
```diff
@@ -227,49 +216,49 @@ If required, increase the `session ID length` by changing the `sessionIDLength`
...
 
 ```java 
-log4j.appender.CARBON_LOGFILE=org.apache.log4j.RollingFileAppender
-log4j.appender.CARBON_LOGFILE=${carbon.home}/repository/logs/${instance.log}/wso2carbon${instance.log}.log
-log4j.appender.CARBON_LOGFILE.MaxFileSize=1000KB
-log4j.appender.CARBON_LOGFILE.MaxBackupIndex=10
+appender.CARBON_LOGFILE.type = RollingFile
+appender.CARBON_LOGFILE.fileName = ${sys:carbon.home}/repository/logs/wso2carbon.log
+appender.CARBON_LOGFILE.policies.size.size=10MB
+appender.CARBON_LOGFILE.strategy.max = 20
```
[the remark this answers is not in the document]
Comment (gomathyk) [reply en/docs/deploy/security/product-level-security-guidelines.md:256]: Will resolve this separately since we don't have a document with the updated content 
https://github.com/wso2/product-is/issues/11964

File: en/docs/deploy/security/encrypt-passwords-with-cipher-tool.md (hunk 15 above)
Comment (gomathyk) [reply en/docs/deploy/security/encrypt-passwords-with-cipher-tool.md:66]: Will resolve this separately since we don't have a document with the updated content 
https://github.com/wso2/product-is/issues/11964

Comment (dewniMW) [reply en/docs/deploy/security/encrypt-passwords-with-cipher-tool.md:66]: in that case, shouldn't we hide it?

File: en/docs/deploy/security/product-level-security-guidelines.md (hunk 16 above)
[the remark this answers is not in the document]
Comment (dewniMW) [reply en/docs/deploy/security/product-level-security-guidelines.md:256]: shall we hide it for now?

**Review by dewniMW — APPROVED** [review]
